bin/update-packages-to-stable.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This script updates the base composer.json in Jetpack to point to the latest version of each package.
+# Probably will be most useful in the release scripts that branch off, since we:
+# a. Want to ship Jetpack with specific versions of packages
+# b. Want to preserve @dev in master branch
+#
+# I was getting rate limited by the GH API when testing, so if this happens to you, you'll have to
+# pass in a personal access token from GitHub to get more requests.
+# Example: `bin/get-latest-package.sh -t YOUR_GH_ACCESS_TOKEN`
+while getopts ":t:" opt; do
+	case ${opt} in
+		t ) GH_TOKEN=$OPTARG
+		    AUTH_HEADER="-H 'Authorization: token $GH_TOKEN'"
+			;;
+		? )
+			echo "Invalid argument: $OPTARG"
+			echo ""
+			;;
+		: )
+			AUTH_HEADER=""
+			;;
+	esac
+done
+shift "$(($OPTIND -1))"
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+JETPACK_ROOT="$(dirname "$DIR")"
+PACKAGE_NAMES=$( ls -d $JETPACK_ROOT/packages/*/ )
+
+for PACKAGE_NAME in $PACKAGE_NAMES; do
```

Comment:
It basically just loops through all the directory names in `packages/` to get the names of the packages that need versions. Similar to PHP's glob I guess. Would it be better to parse the composer.json file itself and look for strings that start with `automattic/jetpack-*`?

Reply:
If it did that, it would be much more re-usable and we could do it for each respective sub-package.

Reply:
The script now can update any directory that has a composer.json file in it.